spec/support/support_helper.rb

```diff
@@ -0,0 +1,19 @@
+module SupportHelper
+  def self.included(base)
+    included_ce(base)
+    included_pro(base) if defined?(Dradis::Pro)
```

Comment:
This is a no-op in CE anyway

```suggestion
    included_pro(base)
```